test/utils/rotation/etcd_encryption_key.go

```diff
@@ -157,29 +168,45 @@ func (v *ETCDEncryptionKeyVerifier) AfterPrepared(ctx context.Context) {
 
 // ExpectCompletingStatus is called while waiting for the Completing status.
 func (v *ETCDEncryptionKeyVerifier) ExpectCompletingStatus(g Gomega) {
```

Comment:
Isn't this function only called when there is a complete annotation set on the shoot, which isn't the case when `v.AutoCompleteAfterPrepared` is true: https://github.com/gardener/gardener/blob/390ae1f896eb5ceb977deabd3529169c56fb6243/test/e2e/gardener/shoot/create_rotate_delete.go#L96-L119